WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TopTabView.swift

```diff
@@ -151,7 +151,7 @@ struct TopTabView<Content: View>: View {
             // Display all the tabs in an HStack, each tab the same width as the TopTabView
             // This GeometryReader is used to set the width and drag offsets for swiping between views
             GeometryReader { geometry in
-                HStack(spacing: 0) {
+                HStack(alignment: .top, spacing: 0) {
```

Comment:
This aligns each tab's content to the top of the tab content area. (This doesn't change the UI for tabs where the tab content is the same height, but when the tabs have different heights it ensures every tab is aligned to the top.)